Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace.package]
-version = "0.0.0" # managed by cargo-workspaces, see below
+version = "0.0.0"                               # managed by cargo-workspaces, see below
```

Comment:
some autoformatting seems to have kicked in, best to review with whitespace hidden
the real change in this file is the removal of the mock-node and/or mock_node